albumentations/augmentations/transforms.py

```diff
@@ -3320,3 +3321,56 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class AugMix(ImageOnlyTransform):
+    """Augmentations mix to Improve Robustness and Uncertainty.
+    Args:
+        image (np.ndarray): Raw input image of shape (h, w, c)
+        severity (int): Severity of underlying augmentation operators.
+        width (int): Width of augmentation chain
+        depth (int): Depth of augmentation chain. -1 enables stochastic depth uniformly
+          from [1, 3]
+        alpha (float): Probability coefficient for Beta and Dirichlet distributions.
+        augmentations (list of augmentations): Augmentations that need to mix and perform.
+    Targets:
+        image
+    Image types:
+        uint8, float32
+
+    Reference:
+    |  https://arxiv.org/abs/1912.02781
+    |  https://github.com/google-research/augmix
+    """
+
+    def __init__(self, width=2, depth=2, alpha=0.5, augmentations=[HorizontalFlip()], always_apply=False, p=0.5):
+        super(AugMix, self).__init__(always_apply, p)
+        self.width = width
+        self.depth = depth
+        self.alpha = alpha
+        self.augmentations = augmentations
+        self.ws = np.float32(np.random.dirichlet([self.alpha] * self.width))
+        self.m = np.float32(np.random.beta(self.alpha, self.alpha))
+
+    def apply_op(self, image, op):
+        image = op(image=image)["image"]
+        return image
+
+    def apply(self, img, **params):
+        mix = np.zeros_like(img)
+        for i in range(self.width):
+            image_aug = img.copy()
+
+            for _ in range(self.depth):
+                op = np.random.choice(self.augmentations)
+                image_aug = self.apply_op(image_aug, op)
+
+            mix = np.add(mix, self.ws[i] * image_aug, out=mix, casting="unsafe")
+
+        mixed = (1 - self.m) * img + self.m * mix
+        if img.dtype in ["uint8", "uint16", "uint32", "uint64"]:
+            mixed = np.clip((mixed), 0, 255).astype(np.uint8)
+        return mixed
+
+    def get_transform_init_args_names(self):
+        return ("width", "depth", "alpha")
```

Comment:
looks like augmentations are not here so they would not be processed correctly during serialization. 
this augmentation looks `Compose`-ish so please take a look at `Compose`-like augmentations. 
and you can write test for serialization of this class